charts/opentelemetry-operator/values.yaml

```diff
@@ -29,10 +29,13 @@ pdb:
 manager:
   image:
     repository: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    tag: v0.87.0
+    tag: v0.88.0
   collectorImage:
     repository: otel/opentelemetry-collector-contrib
-    tag: 0.87.0
+    tag: 0.88.0
+  operatorOpampBridgeImage:
+    repository: ""
```

Comment:
Lose or forget to set the default repo and tag value? Because I noticed that you use it with `{{- if and .Values.manager.operatorOpampBridgeImage.repository .Values.manager.operatorOpampBridgeImage.tag }}` in `templates/deployment.yaml`

Reply:
i was following what's done with the target allocator image which is the same empty thing. The operator fills in these with its own repo, image, and tag so it's not necessary unless its meant to be overridden.

Reply:
Leaving these blank means we'll default to the operator's default, which is proper.